README.md

```diff
@@ -6,6 +6,62 @@ This repository contains documentation for the MongoDB Kotlin Driver.
 
 Please file issue reports or requests at the [Documentation Jira Project](https://jira.mongodb.org/browse/DOCS).
 
+## Building API Documentation
+
+Build the API documentation docs for any major and minor version releases.
+
+We use [Dokka](https://github.com/Kotlin/dokka) to generate the API documentation and host them on GitHub Pages.
+
+To execute a build, you must have Java JDK 17+ installed. You can use brew to install jEnv and OpenJDK to manage your Java versions:
+
+```sh
+brew install jenv
+brew install openjdk@17
+```
+
+The output of the `openjdk@17` includes a command that starts with `sudo ln -sfn`. Execute this command and then run the following commands to add the JDK to your environment:
+
+```sh
+jenv add /Library/Java/JavaVirtualMachines/openjdk-17.jdk/Contents/Home/
+jenv global 17.0
+```
+
+Fork and clone the [`mongo-java-driver`](https://github.com/mongodb/mongo-java-driver) repository. Navigate to the repository and run the following `gradle` command:
+
+```sh
+./gradlew dokkaHtml
+```
+
+This should generate the docs in `./build/docs/`
+Check out the upstream `gh-pages` branch:
+
+```sh
+git checkout gh-pages
```

Comment:
Issue:
I think the instructions are missing the step to create a branch off gh-pages.
Suggestion:
Add a step that mentions creating a feature branch.